packages/perspective-node/package.json

```diff
@@ -0,0 +1,52 @@
+{
+  "name": "perspective-node",
```

Comment:
ok, lets remove use of learna then, this doesn't seem complex enough to have a lot of future dev and need for testing packages.